when-shit-hits-the-fan.adoc

```diff
@@ -484,29 +483,29 @@ locks needed exceeds the limit, the block will be considered
 invalid. So if X+1 exceeds the limit but not X, then the two nodes
 will split the blockchain and disagree on which branch is valid.
 
-The solution chosen, apart from the immediate actions by two pools to
+The solution chosen, apart from the immediate actions taken by the two pools to
 restore consensus, was to
 
 * limit the blocks in terms of both size and locks needed on version
   0.8.1
-* patch old versions, 0.7.2 and some older ones, with the same new
-  rules and increase the global lock limit.
+* patch old versions (0.7.2 and some older ones) with the same new
+  rules, and increase the lock limit.
```

Comment:
I think the word global is quite important here. There's a global lock limit for the system as a whole, and a per-block lock limit. Can we add it back? Please?

Reply:
I didn't catch it was a specific technical term, sorry about that